### PR TITLE
Introduced automation-info in content-form

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | https://github.com/massiveart/husky/pull/756 https://github.com/sulu/SuluArticleBundle/pull/55
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR introduces an info field under the save-button to show upcoming tasks.

#### TODO

- [x] Only if automation-bundle exists
- [x] Translations (FR)
- [x] Test TaskController
- [x] ArticleBundle

### CODE REVIEW

File: src/Sulu/Bundle/AutomationBundle/Resources/public/js/components/automation-tab/main.js [hunk 1]
```diff
@@ -220,6 +228,7 @@ define([
                 _.each(ids, function(id) {
                     --this.notificationBadge;
                     this.sandbox.emit('husky.datagrid.tasks.record.remove', id);
+                    this.sandbox.emit('automation.task.remove', id);
```
Comment (danrot) [inline src/Sulu/Bundle/AutomationBundle/Resources/public/js/components/automation-tab/main.js]: Don't you miss a `sulu` prefix here?

File: src/Sulu/Bundle/AutomationBundle/Resources/public/js/components/automation-tab/main.js [hunk 2]
```diff
@@ -238,12 +247,15 @@ define([
             data.entityId = this.entityData[this.options.idKey];
 
             return manager.save(data).then(function(response) {
-                var event = 'husky.datagrid.tasks.record.add';
+                var datagridEvent = 'husky.datagrid.tasks.record.add';
+                var automationEvent = 'automation.task.create';
```
Comment (danrot) [inline src/Sulu/Bundle/AutomationBundle/Resources/public/js/components/automation-tab/main.js]: Make a single `var` statement.

File: src/Sulu/Bundle/AutomationBundle/Resources/public/js/main.js [hunk 3]
```diff
@@ -25,7 +25,11 @@ define(['css!suluautomationcss/main'], function() {
 
             'use strict';
 
+            config.set('sulu_automation', true);
```
Comment (danrot) [inline src/Sulu/Bundle/AutomationBundle/Resources/public/js/main.js]: Shouldn't we use something like `sulu_automation.enabled` or something like this? This way it is hard to add more config options later.

**Comment by wachterjohannes** [comment]
@danrot fixed

**Comment by danrot** [comment]
As you can see in the following screenshot it is showing the wrong task to me:

image

**Comment by wachterjohannes** [comment]
@danrot missing sortBy -> is fixed now

**Pull request merged**